ocs_ci/ocs/defaults.py

```diff
@@ -84,5 +84,6 @@
         "VAULT_CACERT_FILE": "fullchain.pem",
         "VAULT_CLIENT_CERT_FILE": "cert.pem",
         "VAULT_CLIENT_KEY_FILE": "privkey.pem",
+        "VAULT_BACKEND": "kv-v2",
```

Comment:
Wouldn't it be better to leave "1-vault" for v1 and create "2-vault" for v2?
By the way I used "VAULT_BACKEND: v2", does "kv-v2" works also in configmap?

Reply:
In the test case, this is replaced by custom key names like `vault-test-xxxxxxxx`, so that there are no conflicts with any existing keys in the configmap. 
For PV encryption `kv` and `kv-v2` are the the values that ceph-csi recognises. (https://bugzilla.redhat.com/show_bug.cgi?id=1983931#c16)